content/doc/developer/book.html.haml

```diff
@@ -33,5 +33,5 @@ layout: developerguide
                 %i.icon-warning{:title => 'Partial work in progress', :style => 'opacity: 0.5;'}
                   &nbsp;
   %li{:style => 'font-size: 24px;'}
-    %a{:href => 'guides'}
+    %a{:href => '../guides'}
```

Comment:
Does not seem to be related

Reply:
yep, but i was too lazy to create separate PR for this fix